components/infobox/wikis/stormgate/infobox_skill_custom.lua

```diff
@@ -109,10 +113,20 @@ function CustomInjector:parse(id, widgets)
 				Cell{name = 'Effect', content = makeArrayLinks(Array.parseCommaSeparatedString(args.effect), ' %(effect%)$')},
 				Cell{name = 'Trigger', content = {args.trigger}},
 				Cell{name = 'Invulnerable', content = makeArrayLinks(Array.parseCommaSeparatedString(args.invulnerable))},
+				Cell{name = 'Introduced', content = {introduced}}
 			},
 			caller:_damageHealDisplay('damage'),
 			caller:_damageHealDisplay('heal')
 		)
+		if args.deprecated then
+			local patch = '[['.. CustomSkill._getPatchName(args.deprecated) .. '|' .. args.deprecated .. ']]'
+			local box = messageBox.main( 'ambox', {
```

Comment:
not sure i am a fan of using am box inside infobox
maybe below it would be better

Reply:
As part of a new Module/Template like its done with `{{CardList}}` for Upgrades and Abilities?

Reply:
no i was thinking of just appending it at the end of the infobox but not as bottom content

basically at the end of the run function 
```lua
return mw.html.create()
	:node(builtInfobox)
	:node(warningBoxCall)
```